_includes/integrations/react-quick-start.md

```diff
@@ -18,69 +18,65 @@ This procedure requires:
 
 ### Procedure
 
-1. On a command line or command prompt, install the [Create React App](https://github.com/facebook/create-react-app) package.
+1. Use the the [Create React App](https://github.com/facebook/create-react-app) package to create a new React project named `tinymce-react-demo`.
 
     ```sh
-    $ npm install -g create-react-app
+    $ npx create-react-app tinymce-react-demo
     ```
 
-2. Create a new React project named `tinymce-react-demo`.
-
-    ```sh
-    $ create-react-app tinymce-react-demo
-    ```
-
-3. Change into the newly created directory.
+2. Change into the newly created directory.
 
     ```sh
     $ cd tinymce-react-demo
     ```
 
-4. Install the `tinymce-react` package and save it to your `package.json` with `--save`.
+3. Install the `tinymce-react` package and save it to your `package.json` with `--save`.
 
     ```sh
     $ npm install --save @tinymce/tinymce-react
     ```
 
-5. Using a text editor, open `/path/to/tinymce-react-demo/src/App.js` and replace the contents with:
+4. Using a text editor, open `./src/App.js` and replace the contents with:
```

Comment:
This won't work verbatim on Windows, do we care? I don't think so but just wanted to double check.

Reply:
It's maybe not the most natural path for windows but it still accepts it in powershell...